content/en/docs/setup/pick-right-solution.md

```diff
@@ -34,6 +34,8 @@ a Kubernetes cluster from scratch.
 
 * [Minikube](/docs/setup/minikube/) is a method for creating a local, single-node Kubernetes cluster for development and testing. Setup is completely automated and doesn't require a cloud provider account.
 
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) is an easy-to-install application for your Mac or Windows environment that enables you to start coding and containerizing in minutes on a single-node Kubernetes cluster.
```

Comment:
Just a couple nits:

- Can you please wrap lines at 80 characters. I know the rest of the file isn't that way currently, but it really helps for granular reviews.

- Can you change "containerizing" to "creating containers" or something similar? We have lots of readers who might not understand that word, and it might be hard to translate into other languages.

Reply:
i went for "deploying in containers", let me know if that doesn't sound too awkward.